phonon/update.py

```diff
@@ -236,3 +230,117 @@ def merge(self, update):
         """
         raise NotImplemented("You must define a merge method that merges it's\
             argument with this object.")
+
+class Update(BaseUpdate):
+
+    def _BaseUpdate__merge(self):
+        pickled = self._BaseUpdate__process.client.get(self.resource_id)
```

Comment:
How would you feel about the private methods in the `BaseUpdate` class becoming 'protected' methods? Ideally, the overwritten methods would be accessible from within the base class to avoid code redundancy between the `Update` and `ConflictFreeUpdate` classes. However, as long as they are private, there will be issues with the inheritance.